src/bundle.js

```diff
@@ -37,7 +37,7 @@ const fs = require('fs-extra')
  * @param {Function} [log] the app logger
  * @returns {BundleWebObject} the BundleWebObject
  */
-module.exports = async (entryFile, dest, options = {}, log = () => {}) => {
+module.exports = async (entryFile, dest, options = { shouldOptimize: false }, log = () => {}) => {
```

Comment:
I realize this will leave a residue when we append `...options` below but i guess it's ok.